apps/integration_tests/log_event_schemas.py

```diff
@@ -97,6 +97,7 @@
         "auth_app_data_access_type": {"pattern": "RESEARCH_STUDY"},
         "auth_require_demographic_scopes": {"pattern": "^True$"},
         "req_qparam_client_id": {"type": "string"},
+        "req_qparam_lang": {"type": "string"},

```

Comment:
This being required is breaking unit tests somewhere. See the PR actions for details, you may be able to recreate locally. Not sure if the fix would be to update the tests or to make this not required, but I'll leave that for you to figure out.